packages/client/src/components/FooterSection/FooterSection.component.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './FooterSection.style.css';
+import { Link } from "react-router-dom";
+import vectorFacebook from '../../../public/assets/vectors/vectorFacebook.svg';
+import vectorInstagram from '../../../public/assets/vectors/vectorInstagram.svg';
+import vectorLinkedin from '../../../public/assets/vectors/vectorLinkedin.svg';
```

Comment:
Everything under `public` should be served by the web server (both the main app and Storybook), so you shouldn't import these files. 

Instead you should reference them with an absolute URL like `src="/assets/vectors/vectorLinkedin.svg"`.